homeassistant/components/plugwise/sensor.py

```diff
@@ -301,6 +301,22 @@
         entity_category=EntityCategory.DIAGNOSTIC,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    SensorEntityDescription(
+        key="domestic_hot_water_setpoint",
+        name="DHW setpoint",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="maximum_boiler_temperature",
+        name="Maximum boiler temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
```

Comment:
The PR description says:

> When not a number, show as sensor (when available)

However, this always creates the sensor it is not a replacement/instead of the number, right?

Reply:
This only creates the sensor when it's in the data as a sensor and not as a number:
Sensor: `"sensor_name": value`
Number: `"number_name": {"setpoint": value, ...}`

I've made sure that the library never output's both.